modules/checking-mco-status.adoc

```diff
@@ -98,12 +98,12 @@ ExecStart=/usr/bin/hyperkube \
 If something goes wrong with a machineconfig that you apply, you can always
 back out that change. For example, if you had run `oc create -f ./myconfig.yaml`
 to apply a machineconfig, you could remove that machineconfig by typing:
-+
```

Comment:
I don't know asciidoc well; is removing this intentional?

Reply:
The change itself looks like Git being a pain about something, but the syntax looks right.

Reply:
I saw the spurious characters in the render, so I removed them

![image](https://user-images.githubusercontent.com/10236257/100928697-89778080-34b4-11eb-8bbe-9604d6f93a2e.png)